misc/theme-cicada/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=		theme-cicada
+PLUGIN_VERSION=		0.2
+PLUGIN_REVISION=	2
+PLUGIN_COMMENT=		The cicada theme - grey/orange
+PLUGIN_MAINTAINER=	remic-webdesign@chello.at
```

Comment:
@marjohn56 If you push it, aren't you the maintainer?

Reply:
Very good question, to which I do not know the simple answer. ;)

I'm just the pipe... I'll change this to me and credit remic-*** in the comment.

Reply:
Yes please, we can pass it over later during a 1.x transition when he handles the PRs.